src/components/shared/NewEventForm.js

```diff
@@ -0,0 +1,107 @@
+import React from 'react';
+import Button from '@material-ui/core/Button';
+import Dialog from '@material-ui/core/Dialog';
+import DialogActions from '@material-ui/core/DialogActions';
+import DialogContent from '@material-ui/core/DialogContent';
+
+import DialogTitle from '@material-ui/core/DialogTitle';
+
+import Fab from '@material-ui/core/Fab';
+import AddIcon from '@material-ui/icons/Add';
+import Tooltip from '@material-ui/core/Tooltip';
+
+import TextField from '@material-ui/core/TextField';
+import Grid from '@material-ui/core/Grid';
+import AccountCircle from '@material-ui/icons/AccountCircle';
+import { makeStyles } from '@material-ui/core';
+
+const useStyles = makeStyles({
+  root: {
+    margin: '30%',
+    padding: '30%'
+  }
+});
+
+export default function NewEventForm() {
+  const [open, setOpen] = React.useState(false);
+  const classes = useStyles();
+  const handleClickOpen = () => {
+    setOpen(true);
+  };
+
+  const handleClose = () => {
+    setOpen(false);
+  };
+
+  return (
+    <div>
+      <Tooltip
+        title="Add New Event"
+        onClick={handleClickOpen}
```

Comment:
I think the better way of doing it is `onClick={ () => setOpen(true) }` for simplicity. And the same thing for `onClose={handleClose}`